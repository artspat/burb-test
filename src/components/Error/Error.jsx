import React from 'react';
import './Error.scss';

export const Error = ({ text }) => (
    <span className="b-error">{text}</span>
);