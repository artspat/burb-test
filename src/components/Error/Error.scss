@import '../../styles/global.scss';

.b-error {
    color: $red;
}
