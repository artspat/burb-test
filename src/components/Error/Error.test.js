import React from 'react';
import { shallow } from 'enzyme';
import { Error } from './Error';

describe('<Error />', () => {
    describe('when props are passed', () => {
        it('should render component', () => {
            const wrapper = shallow(<Error text="error text" />);
            expect(wrapper).toMatchSnapshot();
        });
    });
});
