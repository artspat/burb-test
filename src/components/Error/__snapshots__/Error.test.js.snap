// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> when props are passed should render component 1`] = `
<span
  className="b-error"
>
  error text
</span>
`;
